gallery/gallery.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Image Gallery</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/4.1.1/animate.min.css" />
+    <link rel="stylesheet" href="../style.css" />
+    <link rel="stylesheet" href="./gallery-styles.css" />
+  </head>
+  <body>
+    <header>
+      <div class="nav-container">
```

Comment:
didn't see any css for `.nav-container` which is fine but if you're not styling it, it seems redundant when you're using the nav element.